examples/nextjs-comments-video/src/app/api/webhooks/route.ts

```diff
@@ -0,0 +1,66 @@
+import { NextRequest, NextResponse } from "next/server";
```

Comment:
Do we want webhooks in this example specifically?

Reply:
Whoops, no, mistake!